test/sql/utils/testsupport.sql

```diff
@@ -280,3 +280,14 @@ BEGIN
 END
 $BODY$;
 
+CREATE OR REPLACE FUNCTION test.make_tablespace_path(prefix TEXT, test_name TEXT)
+       RETURNS TEXT LANGUAGE plpgsql AS
+$BODY$
+DECLARE
+    dirPath TEXT := format('%s%s', prefix, test_name);
+    createDir TEXT := format('mkdir %s', dirPath);
```

Comment:
`mkdir -p` is not necessary here? Does this test supposed to run on Windows, I wonder how it will be handled then

Reply:
This test is supposed to run on Windows, yes, which is why I didn't add the `-p` flag. The prefix path is constructed on Windows and Linux before the tests are started, so this will just create a sub-directory under the prefix.